entry_point.py

```diff
@@ -59,6 +61,66 @@ def change_values(file_name, getter_func):
         print(new_str.replace('\n', ''))
 
 
+def get_varname(var, prefix="ODOORC_"):
+    """
+    Cleans the string and return the variable ready to use in the config file
+
+    :param var: variable name as comes from the source
+    :param prefix: prefix to be deleted from the string
+    :return: Sting in lowercase without the prefix
+    """
+    r = re.compile("^"+prefix)
+    return r.sub("", var, 1).lower()
+
+
+def get_odoo_vars(getter_func):
+    res = dict()
+    for key, value in getter_func():
+        if key.startswith("ODOORC_"):
+            var = get_varname(key)
+            res.update({var: value.strip()})
```

Comment:
```suggestion
            res.update({get_varname(k): v.strip() for k, v in getter_func() if k.startswith('ODOORC_')})
```   

Something like this can be used instead of the whole method. Or you could even remove the `get_varname` method and make this method like:   

```python
def get_odoo_vars(prefix='ODOORC_'):
    res = {}
    r = re.compile("^"+prefix)
    res.update({r.sub("", k, 1).lower(): v.strip() for k, v in getter_func() if k.startswith(prefix)})
```      
This way you also avoid hardcoding `ODOORC_` in the `startswith` method and leave it in a mor generic way in case you want to use variables like `BASHRC_` in the future.     

![image](https://user-images.githubusercontent.com/12100691/61068775-697f8280-a3d0-11e9-867a-878d3a792fa8.png)

Reply:
@Angelfentanez Because is easier to read, maintain, test and no one in the future will be asking me why do I code so cryptically :/

Reply:
![image](https://user-images.githubusercontent.com/12100691/61070261-dd6f5a00-a3d3-11e9-9bc6-d71ee2b67aa9.png)